src/widgets/attachments/image-preview.component.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import styles from "./image-preview.css";
+
+export default function ImagePreview(props: ImagePreviewProps) {
+  const [caption, setCaption] = useState("");
+
+  function saveImage(e: React.SyntheticEvent) {
+    e.preventDefault();
+    props.onSaveImage(props.dataUri, caption);
+  }
+
+  function cancelCapture(e: React.SyntheticEvent) {
+    e.preventDefault();
+    props.onCancelCapture();
+  }
+
+  function handleSubmit(e: React.SyntheticEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+  }
+
+  function updateCaption(e: React.SyntheticEvent) {
```

Comment:
Prefer to use `React.ChangeEvent<HTMLInputElement>`. Then we also don't need to cast below.